Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian
+
+RUN apt-get update
+
+RUN apt-get install -y \
+  lua5.2 \
+  liblua5.2-dev \
+  luarocks \
+  git \
+  libssl1.0-dev \
+  make
+
+RUN luarocks install --server=http://luarocks.org/dev luaffi
```

Comment:
Why do we need this ? If it's a dependency we should have listed in Makefile.

Reply:
Unfortunately my lua knowledge isn't much better. But it really does not work without it.

If you could then please add it to the Makefile I would be happy to approve.